server/main.go

```diff
@@ -30,6 +32,9 @@ var (
 	port                  = flag.String("port", ":8080", "port to listen on")
 	databasePath          = flag.String("db", "./db/development.db", "path to sqlite3 database file")
 	templatesPath         = flag.String("templates", "./assets/templates", "path to template files")
+	env                   = flag.String("env", "production", "environment applikatoni is used in")
+	dbConfDir             = flag.String("dbconfdir", "./db", "path to dbconf.yml file")
```

Comment:
What about `"path to directory of dbconf.yml"` file? Compared to `"path to sqlite3 database file"` the `"path to dbconf.yml file"` could be confusing. The former really needs the path to a _file_ and the latter a path or _directory_.